Python/executor_cases.c.h

```diff
@@ -1637,11 +1813,10 @@
             break;
         }
 
-        case _LOAD_ATTR_INSTANCE_VALUE: {
+        case _LOAD_ATTR_INSTANCE_VALUE_0: {
             PyObject *owner;
             PyObject *attr;
             PyObject *null = NULL;
```

Comment:
Ugh, because `null` is not used in the `oparg == 0` case, this now causes a GitHub warning. Not sure how to fix without a lot of extra code in the generator -- in handwritten code we'd use `(void)null;` .

Reply:
Fixed